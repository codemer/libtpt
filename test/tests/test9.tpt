@set(x, 10)\
@set(y, 2)\
@set(z, 9)\
@set(result, (x+y)*z)\
x=${x}
y=${y}
z=${z}
(x+y)*z=${result}

