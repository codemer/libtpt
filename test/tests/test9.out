@set(x, 10)\
@set(y, 2)\
@set(z, 9)\
@set(result, (x+y)*z)\
x=10
y=2
z=9
(x+y)*z=108

